pkgs/racket-doc/scribblings/reference/treelists.scrbl

```diff
@@ -54,7 +54,7 @@ Returns a @tech{treelist} with @racket[v]s as its elements in order.
 
 This operation takes @math{O(N log N)} time to construct a treelist of
 @math{N} elements. Unless otherwise specified, other operations on a
-treelist of length @math{N} also take @math{O(log N)} time.
+treelist of length @math{N} take @math{O(log N)} time.
```

Comment:
It seems weird to document the running time of "other operations" in this `treelist` function. I think the introductory paragraph would work better.

Reply:
I moved it, since I had put this here in the first place.